Source/Resource/boolean_profiles/default.json

```diff
@@ -0,0 +1,775 @@
+{
+    "conditions": [
+        {
+            "group": 0,
+            "plotConditions": [
+                {
+                    "curveConditions": [
```

Comment:
I suggest `curveConditions` to `formulas`. Also suggest changing `curve*` to `formula*` throughout.

Reply:
ok.

Reply:
done